### ISSUE TITLE
Override style of element (without editing lib's css)?

### ISSUE THREAD

**Issue by MatiasSMd** [isteven/angular-multi-select#331]
I have successfully integrated an isteven-multi-select in my application. However I can not find a way of changing the style of the main button (without copying and rewriting the css distributed as part of the your project). 
Is there any recommended way of accomplishing this? Did you considered adding an "extension" point like the one discussed in [this post of SO](http://stackoverflow.com/questions/19246110/how-to-manipulate-styles-of-directive-in-angularjs#answer-19250312)?

PS: the reluctance to editing the css is to ease lib upgrade

**Comment by isteven** [comment]
Hi @MatiasSMd ,

I understand your concern (ease of upgrade), but as for now I couldn't really think of a better way.

Long ago I actually did consider custom CSS class names to be passed into directive like that SO post, but then again with limited time to code, I chose to work on more important functionalities. I also found that most users don't really mind to edit the CSS file.

What I can say is actually, at this point of time, there won't be big changes in CSS, so I can say that it's quite safe to edit & play with it.
